documentation/asciidoc/accessories/display/legacy.adoc

```diff
@@ -1,6 +1,31 @@
 == Legacy Support
 
-NOTE: These instructions are for the original Raspberry Pi 1 Model A and B boards only. You can identify an original board as it is the only model with a 26-pin GPIO header, all other models have the now-standard 40-pin connector.
+[NOTE]
+====
+Older Raspberry Pi hardware, and older versions of Raspberry Pi OS, with X11 can install an on-screen keyboard and emulate right clicks on the touchscreen. 
+
+Run the following in a terminal to enable the keyboard:
+----
+sudo apt install matchbox-keyboard
+----
+
+Add the following to `/etc/X11/xorg.conf` enable right click emulation:
```

Comment:
Missing "to" before "enable".
@aallan Should "right click emulation" be "right-click emulation" ?

Reply:
> @aallan Should "right click emulation" be "right-click emulation" ?

Yes.

Reply:
@jehlers42 https://github.com/raspberrypi/style-guide/blob/master/style-guide.md says `right-click`, so that's probably the green-light you need :vertical_traffic_light: :smiley: 
(although probably "cleaner" to do it as a separate PR, rather than bundling it into this PR)